CanonizerRegistrationPage.py

```diff
@@ -0,0 +1,196 @@
+    """
+    Class Name: CanonizerRegisterPage
+
+    """
+
+    def click_register_button(self):
+        """
+        -> Hover the control towards the register button. Identifiers are loaded from Identifiers Class
+        -> Find the register button and Click on it.
+
+        :return:
+            Return the control to the main Program
+        """
+
+        self.hover(*RegistrationPageIdentifiers.REGISTER)
+        self.find_element(*RegistrationPageIdentifiers.REGISTER).click()
+
+        return CanonizerRegisterPage(self.driver)
+
+    def register_page_mandatory_fields_are_marked_with_asterisk(self):
+        """
+        This Function checks, if Mandatory fields on Register Page Marked with *
+        First Name, Last Name, Email, Password, Confirm Password are Mandatory Fields
+
+        :return: the element value
+        """
+        return \
+            self.find_element(*RegistrationPageIdentifiers.FIRST_NAME_ASTRK) and \
+            self.find_element(*RegistrationPageIdentifiers.LAST_NAME_ASTRK) and \
+            self.find_element(*RegistrationPageIdentifiers.EMAIL_ASTRK) and \
+            self.find_element(*RegistrationPageIdentifiers.PASSWORD_ASTRK) and \
+            self.find_element(*RegistrationPageIdentifiers.CONFIRM_PASSWORD_ASTRK)
+
+    def enter_first_name(self, firstname):
+        self.find_element(*RegistrationPageIdentifiers.FIRST_NAME).send_keys(firstname)
+
+    def enter_last_name(self, lastname, ):
+        self.find_element(*RegistrationPageIdentifiers.LAST_NAME).send_keys(lastname)
+
+    def enter_email(self, user):
+        self.find_element(*RegistrationPageIdentifiers.EMAIL).send_keys(user)
+
+    def enter_mobile_number(self, mobile_number):
+        self.find_element(*RegistrationPageIdentifiers.MOBILE_NUMBER).send_keys(mobile_number)
+
+    def enter_password(self, password):
+        self.find_element(*RegistrationPageIdentifiers.PASSWORD).send_keys(password)
+
+    def enter_confirm_password(self, confirmpassword):
+        self.find_element(*RegistrationPageIdentifiers.CONFIRM_PASSWORD).send_keys(confirmpassword)
+
+    def click_register_now_button(self):
+        try:
+            WebDriverWait(self.driver, 5).until(
+                EC.visibility_of_element_located(
+                    (By.CLASS_NAME, 'ant-btn ant-btn-primary ant-btn-block login-form-button')))
+        except TimeoutException:
+            pass
+        self.find_element(*RegistrationPageIdentifiers.REGISTER_NOW).click()
+
+    def register(self, *args):
+        self.enter_first_name(args[0])
+        self.enter_last_name(args[1])
+        self.enter_email(args[2])
+        self.enter_mobile_number(args[3])
+        self.enter_password(args[4])
+        self.enter_confirm_password(args[5])
+        self.click_register_now_button()
+
+    def registration_with_blank_first_name(self, REG_LIST_3):
+        self.register(REG_LIST_3[0], REG_LIST_3[1], REG_LIST_3[2], REG_LIST_3[3], REG_LIST_3[4], REG_LIST_3[5])
```

Comment:
Line 82-87 are repeated in line 114-119. Please create a function and then call it